Demo/README.md

```diff
@@ -44,7 +44,9 @@ This example is used to support Drop-in gift card flow.
 ### UIKit Demo
 
 This is the basic UI for UIKit integration.
+You can run it by selecting `AdyenUIHost` target in xcode.
 
 ### SwiftUI Demo
 
 This is the basic UI for SwiftUI integration.
+You can run it by selecting `AdyenSwiftUIHost` target in xcode.
```

Comment:
```suggestion
Run the SwiftUI Demo it by selecting `AdyenSwiftUIHost` as the target in Xcode.
```
Same as above, do users need to do anything else after selecting the target?

Reply:
I guess it should be clear from this point how to run the project.